docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+
+#### Metamask 
 
-Make sure you have at least Node.js `v8.0.0`.
+> MetaMask is a browser plugin that allows users to make Ethereum transactions through regular websites. It facilitates the adoption of Ethereum because it bridges the gap between the user interfaces for Ethereum (eg. Mist browsers, Dapps) and the regular web (eg. Chrome, Firefox, websites).
 
-### Signing and web3 provider
+> MetaMask injects a javascript library called web3.js into the namespace of each page your browser loads. web3.js is written by the Ethereum core team, and has functions that regular webpages can use to make read and write requests on the blockchain that are consistent with the existing protocol [Source](https://www.quora.com/What-is-MetaMask)
 
-We recommend using [Frame](https://frame.sh) to send transactions to the blockchain. Otherwise you can use [Metamask browser extension](https://metamask.io/).
+[^ explain what this means]
 
-### Install the aragonCLI
+If this is your first time using Metamask, here's how to started:
 
-From the command line run:
+Visit https://metamask.io/ and download the relevant browser extension. Once it's downloaded you should be automatically directed to a page that looks like the image below.
+
+![](/docs/assets/metamask-1.png)
+
+Click on the **Get Started** button and follow the instructions carefully. They should be pretty self-explanatory. Once you've finished setting up Metamask, you should be automatically redirected to a page that looks like the one below.
+
+![](/docs/assets/metamask-4.png)
+
+This is the Metamask interface to your Ethereum wallet. Congratulations! You're now ready to interact directly with the Ethereum blockchain. 
+
+#### Frame 
+> Frame is an OS-level Ethereum interface that lets you use standalone signers, such as a Ledger or Trezor, to interact with dapps and the Ethereum network.
+
+[explain what this means and go through installation -- with images]
+
+
+### The aragonCLI
+
+The final missing piece is what's known as the aragonCLI (or Command Line Interface). This is what we'll use to create and develop Aragon apps.
+
+To install it from the command line run:
 
 ```sh
 npm i -g @aragon/cli
 ```
 
-Once we have this package installed we can start building DAOs.
+Hopefully, it downloaded successfully. If that's the case, congrats! you're now officially ready to start building DAOs!
 
-### Note on Git
+If you're having trouble with this step, you should read through the relevant operating system section below. If that doesn't fix things, please don't hesitate to reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help).
+
+### Mac considerations
```

Comment:
What you think of move all this to the [troubleshooting guide](https://hack.aragon.org/docs/guides-faq) under `Installing aragonCLI` section

Reply:
Agree. Much cleaner

Reply:
Moved OS considerations + Note on Git to troubleshooting guide under a new Installing aragonCLI section